packages/fcl/src/current-user/index.js

```diff
@@ -101,7 +101,7 @@ async function makeMessage() {
   }
 }
 
-async function authenticate({service, redir = false}) {
+async function authenticate({service, redir = false} = {}) {
```

Comment:
Does this work? The redirect already has the default val
`authenticate({service = {}, redir = false}) {}`

Reply:
@gregsantos It's that `authenticate()` passes no object to destructure from and we'd like to keep it from having to pass one. It's like doing `let {service} = undefined` and the fix of `= {}` sets an object as the default if none is passed in.